business_rules/utils.py

```diff
@@ -109,6 +109,13 @@ def check_params_valid_for_method(method, given_params, method_type_name):
     defined_params = [param.get('name') for param in method_params]
     missing_params = set(defined_params).difference(given_params)
 
+    # check for default value, if it is present, exclude param from missing params
```

Comment:
Could you specify that this check only is useful for actions and not for variables?